mitodl/hugo-course-publisher
# mobile view for home page

#### Pre-Flight checklist



#### What are the relevant tickets?

https://trello.com/c/ddNpXeoV/23-mobile-home-page

#### What's this PR do?

this basically gets the mobile view of the home page together, implementing what is shown in the mocks.

I also did a few things that apply to the desktop version, like removing the breadcrumbs (which Abdul said we're not doing anymore).

#### How should this be manually tested?

take a look at the designs for the mobile home page, and see if I missed something :)

then also I think check out some pages on desktop to make sure I didn't make any regressions.

#### Screenshots (if appropriate)

![Screenshot from 2020-02-24 13-39-05](https://user-images.githubusercontent.com/6207644/75180872-0e09c700-570b-11ea-9952-ce13973cb56d.png)
![Screenshot from 2020-02-24 13-38-42](https://user-images.githubusercontent.com/6207644/75180873-0e09c700-570b-11ea-9562-ba5c95b281b3.png)
